app/views/declaration/taric_codes.scala.html

```diff
@@ -43,43 +60,67 @@
 
 @tradeTariffLink = {<a target="_blank" href=@appConfig.tradeTariffUrl>@messages("declaration.taricAdditionalCodes.header.tradeTariff.link")</a>}
 
-@main_template(
-    title = Title("declaration.taricAdditionalCodes.header", "supplementary.items")
-) {
+@pageHint = {
+    <span class="govuk-hint">
+        @Html(messages("declaration.taricAdditionalCodes.header.hint", tradeTariffLink))
+    </span>
+}
 
-    @helper.form(routes.TaricCodeController.submitForm(mode, itemId), 'autocomplete -> "off") {
-        @helper.CSRF.formField
+@taricCodesTable = {
```

Comment:
Have you considered making this a component? Similar pattern is repeated in several places.

Reply:
Possibly.   But by the time you've discovered all of the similar places where that component could be re-used it's often simpler not to bother.